### ISSUE TITLE
Framework: Create Updated GCC Serial PR Testing Build

### ISSUE THREAD

**Issue by ZUUL42** [trilinos/Trilinos#6302]
## Enhancement
@trilinos/framework

Look into creating an updated GCC Serial PR Testing build.
The current PR testing serial build uses GCC 4.9.3.
Is having a 4.9.3 still useful or would moving the serial build to GCC 8.3 or 9.2 be better?
Results from #6295, Creating a GCC 8.3 Werror Build, may help direct this decision.

**Comment by ZUUL42** [comment]
This is taken care of with #8081

**Comment by cgcgcg** [comment]
Is it possible that HDF5 is not correctly enabled for this build? See https://testing.sandia.gov/cdash/build/7974771/configure

**Comment by ZUUL42** [comment]
@cgcgcg it appears to be not finding the HDF5 TPL at all and I noticed it built on a cloud node that time.
When I setup the 7.2.0 serial build, I updated the new script we're using to mirror the TPLs to the cloud nodes. Let me double check it and run it to see if that takes care of the issue.
It appears 8106 is building on a machine this time, so it should go through, but I'll still put it on my list of things to take a look at tomorrow.
